### ISSUE TITLE
Feature Request: Using build disabled images for imageName selection

### ISSUE THREAD

**Issue by SarguelUnda** [devspace-cloud/devspace#1000]
**Is your feature request related to a problem?**  
Yes. At the moment it is impossible to use images name definied in devspace.yaml but not builded as an imageName selector in the dev section of the file ( port forwarding sync or logging ).

**Which solution do you suggest?**  

Authorize images name defined in devspace.yaml but not builded as an imageName selector in the dev section

**Which alternative solutions exist?**  

At the moment I use labelSelector for port forwading/sync. It is not supported for logs to my knowledge as it exepct an images list.

```yaml
   - labelSelector:
      app.kubernetes.io/name: devspace-app
      app.kubernetes.io/component: app-frontend # name used in the deployment section
```

**Additional context**  

Inspired from the doc an untested exemple that should fail.
```yaml
images:
  frontend:
    image: dscr.io/${DEVSPACE_USERNAME}/appfrontend
    build:                     # <--- Breaking change
      disabled: true
  backend:
    image: john/appbackend
  database:
    image: john/database
deployments:
- name: app-frontend
  helm:
    componentChart: true
    values:
      containers:
      - image: dscr.io/${DEVSPACE_USERNAME}/appfrontend
- name: app-backend
  helm:
    componentChart: true
    values:
      containers:
      - image: john/appbackend
      - image: john/appbackend-sidecar
- name: app-database
  helm:
    componentChart: true
    values:
      containers:
      - image: john/database
- name: app-cache
  helm:
    componentChart: true
    values:
      containers:
      - image: redis:5.0.5
dev:
  ports:
  - imageName: backend # Will work as usual
    forward:
    - port: 8000
  - imageName: frontend # Won't work
    forward:
    - port: 8080
  logs:
    images:
    - backend
    - frontend # Won't work
```

/kind feature

**Comment by LukasGentele** [comment]
@SarguelUnda Thanks for suggesting this. I think we should definitely implement this. I wasn't even aware of this limitation.

**Comment by FabianKramm** [comment]
@SarguelUnda yes thanks for opening this issue! As discussed we can implement this to have another use case for `build.disabled`

**Comment by LukasGentele** [comment]
Released with v4.8.0.
